pkg/standalone-utils/contracts/ProtocolFeeSweeper.sol

```diff
@@ -60,6 +61,28 @@ contract ProtocolFeeSweeper is IProtocolFeeSweeper, SingletonAuthentication, Ree
         uint256 deadline,
         IProtocolFeeBurner feeBurner
     ) external nonReentrant onlyFeeRecipientOrGovernance {
+        _sweepProtocolFeesForToken(pool, feeToken, minTargetTokenAmountOut, deadline, feeBurner, false);
+    }
+
+    /// @inheritdoc IProtocolFeeSweeper
+    function sweepProtocolFeesForWrappedToken(
+        address pool,
+        IERC20 feeToken,
```

Comment:
```suggestion
        IERC4626 wrappedFeeToken,
```
Setting the type explicitly here would be self-documenting, and doesn't change anything else (you can still pass an IERC4626 to an IERC20 argument)